quickwit/quickwit-search/src/tests.rs

```diff
@@ -666,7 +666,7 @@ async fn test_sort_bm25() {
         let hits: Vec<(f32, u32)> = search_hits("nofreq:two").await;
         assert_eq!(
             &hits[..],
-            &[(0.15965714, 1), (0.12343242, 0), (0.12343242, 2)]
+            &[(0.15965714, 1), (0.12343242, 2), (0.12343242, 0)]
```

Comment:
It really does not matter much. I agree your order is nicer though: if we sort by dates in a descending fashion, we want higher doc_id first.

(timestamp could be tie, but documents tend to be order with time so we want to use them that way to break the tie)
It seems to be the way your code works, so that's ok with me.

Is this order respected both in the merge and the leaf code?